### DISCUSSION TITLE
Using the color code (hex value)

### DISCUSSION THREAD

**Discussion by mary-josephine-1** [buefy/buefy#3984] [Q&A] [answered]
### Overview of the problem

**OS/Browser**: Ubuntu

>  "nuxt": "^2.15.8",
    "nuxt-buefy": "^0.4.24",
    "vee-validate": "^2.2.7",
    "vue": "^2.7.10",

### Description
Do we have an option to change the color of the element using the color code (hex value)?

### Steps to reproduce
`<b-tag colour="#0D022B"> Name </b-tag>`
or
` <b-tag type="#0D022B"> Name </b-tag>`

### Expected behavior
Apply the color of the element using the given color code (hex value).

### Actual behavior
Only default options based on `type` can be selected. Cannot enter color code (hex value).

**Comment by wesdevpro** [comment, marked as answer]
@mary-josephine-1 we don't have a prop that allows you to set the color of the b-tag component. We do not have a prop called type that changes the color of the b-tag. However, those colors are the colors that come out of the box from Bulma(see image below).
![image](https://github.com/buefy/buefy/assets/11604664/965d2986-6b4b-4aef-91a2-1560ca944abd)

You can customize these colors by using CSS or Sass variables. Please see the example in the [Buefy Docs](https://buefy.org/documentation/customization)

You could also use built Bulma Helper classes (e.g has-background-primary) However, these helpers also depend on the Bulma color schema.

As a last resort, you can manually use CSS to customize the color of the tag.

I hope this helps. Please let me know if you have any other further questions.



##### Helpful Links
1. https://bulma.io/documentation/customize/
2. https://bulma.io/documentation/helpers/color-helpers/
3. https://bulma.io/documentation/customize/variables/
4. https://bulma.io/documentation/customize/
5. https://buefy.org/documentation/customization
